### DISCUSSION TITLE
Current Logo Readability

### DISCUSSION THREAD

**Discussion by TheRealDust** [ppy/osu#17430] [Ideas] [answered]
The altered logo is significantly harder to read than the original/stable version, most noticeably on the taskbar. The image blurs, especially on small icon mode, and the colour of the second half of the letters also decreases readability. 
Standard Icon Size (Taskbar):
![image](https://user-images.githubusercontent.com/73678906/159867276-1223eedc-b3f1-40d5-8069-38686ad161c1.png)
Small Icon Size (Taskbar):
![image](https://user-images.githubusercontent.com/73678906/159867516-2d4bf1cd-d266-4423-a9ed-55c7fb9c9da6.png)
Left is stable release logo, right is lazer.

**Comment by peppy** [comment, marked as answer]
What version of windows are you running? It looks fine here:

![Parallels Desktop 2022-03-24 at 08 25 23](https://user-images.githubusercontent.com/191335/159873657-b348a78e-bb90-4010-ad7f-4036732528dc.png)

![Parallels Desktop 2022-03-24 at 08 27 51](https://user-images.githubusercontent.com/191335/159874185-278f96d0-e046-4cc3-aa18-9349b08731a0.png)

![Parallels Desktop 2022-03-24 at 08 28 07](https://user-images.githubusercontent.com/191335/159874251-b8e4f61f-0c18-40ec-b9aa-b86ba90566da.png)

**Reply by peppy** [reply]
Okay never mind, my logos are matching yours when you open them outside of github's stupid scaling algorithms.

The new style is intentional for now (more blurry rather than hard pixels). I'd recommend increasing your taskbar size or DPI. Also for what it's worth, the "small taskbar" is gone in windows 11.
